components/gallery/GalleryItemAction/GalleryItemActionType/GalleryItemBuy.vue

```diff
@@ -6,53 +6,73 @@
       ref="root"
       title="Price"
       :price="nftPrice">
-      <GalleryItemActionSlides
-        v-if="Number(nftPrice)"
-        ref="actionRef"
-        :active="active"
-        :class="{ 'gallery-item-slides-entry': !active }"
-        :disabled="disabled">
-        <template #entry>
-          <NeoTooltip
-            v-if="!active"
-            :active="disabled"
-            :label="$t('tooltip.notEnoughBalance')"
-            append-to-body
-            multiline>
-            <NeoButton
-              :label="label"
-              size="large"
-              class="full-width-action-button"
-              variant="k-accent"
-              :disabled="disabled"
-              no-shadow
-              data-cy="item-buy"
-              @click.native="onClick" />
-          </NeoTooltip>
-        </template>
+      <div v-if="disabled" class="gallery-item-disabled">
+        <NeoTooltip
+          :active="disabled"
+          content-class="buy-tooltip"
+          position="top"
```

Comment:
It's better to set it on the left for the desktop, top on mobile. no need `:auto-close` on mobile

you can use these utils https://github.com/kodadot/nft-gallery/blob/62174ea0b2e7800d2e5d4a6717580959f943d8bb/components/navbar/ExploreDropdown.vue#L2

![image](https://github.com/kodadot/nft-gallery/assets/734428/389664f4-99a3-4930-acd4-7ba4c455c8a0)